README.md

```diff
@@ -17,12 +22,11 @@
   ▪︎
   <a href="https://docs.winglang.io/contributors/handbook">Contribute</a>
 </p>
-
 # Welcome! :wave:
```

Comment:
Can you add this line break back? I think this causes the header to render incorrectly in some places: https://github.com/winglang/wing/blob/269-recognize-community/README.md

Reply:
yeah sure thing, looks like it's broken the welcome text. I've left the badge at the top but maybe it would look nicer someplace else?


<img width="1053" alt="Screenshot 2023-01-02 at 16 42 54" src="https://user-images.githubusercontent.com/60711758/210258977-fb80f428-2cea-4fff-8ae4-685fd6281046.png">

Reply:
I'd be okay omitting the badge or moving it lower in the page, either sounds fine

Reply:
cool, I've removed it for now, didn't think it provided much useful information :D

Reply:
<img width="997" alt="Screenshot 2023-01-02 at 16 52 35" src="https://user-images.githubusercontent.com/60711758/210259900-c826cc0b-001d-40b0-8bda-8712b1334614.png">
everything back to normal :)